docs/iot/iot-partners/index.md

```diff
@@ -0,0 +1,26 @@
+# Tizen IoT with Nubison Cloud
+
+The Nubison cloud can readily be interconnected with Tizen IoT devices, which has an instrinsic capability of seamless connectivity.
+The Nubison cloud empowers the users to send or receive requests for Tizen service applications in real-time with ease.
+
+**Figure: Architecture**
+![Architecture](media/architecture.png)
+
+Compatibility
+- Platform : Tizen 5.0 IoT Headed & Headless and later
+- SDK : Tizen Studio 3.0 and later 
+
+
+## What is Nubison IoT?
+Nubison IoT provides various IoT features and services which can be utilized in various IoT devices. Data anlaysis service is one of distinguished IoT services that Nubison provides.
```

Comment:
@nawab-samsung the word "various" is duplicated in one sentence. How about "diverse IoT devices", or other word instead?

anlaysis -> analysis

Reply:
Yes, @mijin85cho diverse would actually sense better, I am changing it to a simpler word for easy comprehension. 
@bson1012
Kindly change the sentence to: 

Nubison IoT provides various IoT features and services which can be utilized in different IoT devices. Data analysis service is one of distinguished IoT services that Nubison provides. 


@bson1012 I observed that you have unknowingly introduced a few spelling mistakes,  kindly check the spellings, or else instead of writing, kindly do a copy paste. 

In my previous comment "analysis" was correctly spelled, you have unknowingly written it wrongly, while changing the text.

Reply:
done

Reply:
Thanks @bson1012